src/zoo/Animal.java

```diff
@@ -16,4 +20,11 @@ public Animal(String name, int age, boolean isWild) {
     public String giveVoice(){
         return "usual Animal voice ";
     }
+    public String doVildThings(boolean isWild){
+        if(isWild){
+            return "Doing vild things";
+        }else {
+            return "nice " + getClass().getSimpleName();
```

Comment:
please don't use `getClass().getSimpleName()` ;) use `toString` in such cases 
https://stackoverflow.com/questions/36828050/is-it-bad-practice-to-use-object-getclass-in-java

Reply:
yes, never ever :)